CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Direct escape input on Windows using alt
 - Incorrect window size on X11 when waking up from suspend
 - Incorrect width of Unicode 11/12 emojis
+- Lack of error message explaining why shell could not start on Windows (when using Winpty backend)
```

Comment:
This line is a bit long, maybe we can make things a bit more precise and get down to the interesting parts quicker:

```suggestion
- Vague startup crash messages on Windows with WinPTY backend
```